airflow/logging_config.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Could you undo this change as it's the only one in the file please?

Reply:
Sure. I think that one was left after I removed some of the changes from this file.